src/containers/notification/store/mobileSagas.ts

```diff
@@ -0,0 +1,126 @@
+import { all, call, put, select, takeEvery } from 'redux-saga/effects'
+import { waitForBackendSetup } from 'store/backend/sagas'
+import AudiusBackend from 'services/AudiusBackend'
+import {
+  FetchNotificationsSuccessMessage,
+  FetchNotificationsReplaceMessage,
+  FetchNotificationsFailureMessage,
+  ResetNotificationsBadgeCount
+} from 'services/native-mobile-interface/notifications'
+import { getHasAccount } from 'store/account/selectors'
+import { MessageType } from 'services/native-mobile-interface/types'
+import * as notificationActions from './actions'
+import {
+  getNotificationEntities,
+  getNotificationEntity,
+  getNotificationUser,
+  getNotificationUsers
+} from './selectors'
+import { ConnectedNotification, Notification, NotificationType } from './types'
+
+const USER_LENGTH_LIMIT = 7
```

Comment:
This is the number of users to attach to a notification right? 
Could you add a comment.